.evergreen.yml

```diff
@@ -382,15 +386,23 @@ tasks:
       - func: "lint"
   - name: atlas_e2e
     tags: ["e2e"]
+    depends_on:
```

Comment:
since this depends on mcli building let's not trigger this task until we are sure we can build